security/pkg/monitoring/monitoring.go

```diff
@@ -0,0 +1,168 @@
+// Copyright 2019 Istio Authors
```

Comment:
seems this is duplicated https://github.com/istio/istio/blob/master/pilot/pkg/monitoring/monitoring.go, do we plan to move this file to some common pkg ?  /cc @douglas-reid

Reply:
Check the discussion surrounding the previous PR [here](https://github.com/istio/istio/pull/15223)

Reply:
seems @douglas-reid has a wip PR https://github.com/istio/pkg/pull/31, probably better to wait it get in first

Reply:
Sgtm

Reply:
Switched to use the common monitoring package + updated vendor stuff.